addons/web/static/src/webclient/switch_company_menu/switch_company_menu.js

```diff
@@ -109,14 +109,14 @@ export class SwitchCompanyMenu extends Component {
         );
         useChildSubEnv({ companySelector: this.companySelector });
     }
+
+    get isSingleCompany() {
+        return Object.values(this.companyService.allowedCompaniesWithAncestors).length == 1;
```

Comment:
In javascript we usually use strict Equality Check: Use === instead of == for a strict equality check. Using == (loose equality) in JavaScript can lead to unexpected results because it performs type coercion, which means it tries to convert the values being compared to the same type before making the comparison.